2018AsianGamesApp/app/src/main/java/com/example/mrtbkwjn/a2018asiangamesapp/EventAdapter.java

```diff
@@ -0,0 +1,62 @@
+package com.example.mrtbkwjn.a2018asiangamesapp;
+
+
+import android.support.annotation.NonNull;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+import java.util.ArrayList;
+
+
+
+public class EventAdapter extends RecyclerView.Adapter<EventAdapter.EventViewHolder>{
+
+        ArrayList<String> eventList;
+
+        public static class EventViewHolder extends RecyclerView.ViewHolder{
+
+            TextView eventIdText;
+            TextView eventDetailText;
+            TextView eventDate;
+
+
+
```

Comment:
the newline is a bit too much here, please automatically reformat your code 

![2018-06-03 17_29_49-myapplication3 c__users_seiry_documents_github_myapplication3 - _app_src_ma](https://user-images.githubusercontent.com/1240672/40885608-d0506e62-6753-11e8-8727-0cad3b7ba400.png)

Reply:
Noted